dashboard/app/models/sections/section.rb

```diff
@@ -64,6 +64,7 @@ def sti_name
 
   has_many :section_hidden_lessons
   has_many :section_hidden_scripts
+  has_many :teacher_feedbacks
```

Comment:
Do we need this?  If not, maybe we can remove it to reduce the dependencies between the models?

Reply:
Could totally be removed -- mostly added for future developers to see that they could easily get all feedback for a given section (although, per your other comments, this is not entirely true b/c this doesn't work for older teacher feedback). What do you mean by "reduce dependencies"?

Reply:
Good question about "reduce dependencies".  It is my experience that one of the things that contributes to rails apps being monolithic and highly coupled is the overuse of ActiveRecord associations.  I am of the opinion that associations should only be added when needed, but realize that that may not be the pattern here and it's probably something I should check with the broader team on first.  Let's go ahead and add this if that's what's usually done here and I'll follow up separately.  Thanks for the question!

Reply:
I asked in Slack and it sounds like we can omit this if it's not being used.